src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -107,7 +107,8 @@ private void testGeneralQuestionOperations() throws Exception {
         testAjaxOnVisibilityMessageButton();
 
         testDeleteQuestionAction(2);
-        testDeleteQuestionAction(1);
```

Comment:
Will it be possible to add the test without removing an existing test case? Even if the old test case isn't useful, I think it's preferable if they are not removed without good reason in this pr.